rules/integrations/azure/defense_evasion_suppression_rule_created.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2021/08/27"
+maturity = "production"
+updated_date = "2021/08/27"
+integration = "azure"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when a suppression rule is created in Azure. Adversary's could attempt this to evade detection.
```

Comment:
Needs to be ellaborated, some Qs you can aim to answer: suppression rules do what? Affect what?